source/sass/type.scss

```diff
@@ -128,7 +128,7 @@ a {
   font-weight: 400;
   color: #231f20;
   letter-spacing: 0.25px;
-  line-height: 1.2;
+  line-height: 1.3;
```

Comment:
It doesn't really matter for the final generated value but we use  `1.33` in all other places. For consistency let's change it to `1.33` here?